packages/devextreme/js/__internal/grids/grid_core/views/m_grid_view.ts

```diff
@@ -199,11 +199,16 @@ export class ResizingController extends modules.ViewController {
     const totalItemsCount = Math.max(0, this._dataController.totalItemsCount());
     const widgetAriaLabel = this._getWidgetAriaLabel();
     const widgetStatusText = messageLocalization
-      // @ts-expect-error Badly typed format method
+    // @ts-expect-error Badly typed format method
       .format(widgetAriaLabel, totalItemsCount, columnCount);
     // @ts-expect-error Badly typed dxElementWrapper
     const $ariaLabelElement = this.component.$element().children(`.${GRIDBASE_CONTAINER_CLASS}`);
+    // @ts-expect-error Treelist Variable
+    const expandableWidgetAriaLabel = messageLocalization.format(this._expandableWidgetAriaId);
 
+    if (expandableWidgetAriaLabel !== '') {
+      this.component.setAria('roledescription', expandableWidgetAriaLabel, $ariaLabelElement);
```

Comment:
Does aria-label attribute have the same effect here? If so, let's change aria-roledescription to aria-label.

Reply:
Aria-label attribute for this element is validated on some QUnit and e2e test cases. This is the reason why I put the instructions on roledescription instead..